.github/workflows/ci.yml

```diff
@@ -157,6 +157,15 @@ jobs:
       - name: Run sanity check
         run: bash ./build/scripts/sanity-check
 
+      - name: Cache phpunit result cache
+        uses: actions/cache@v2
+        with:
+          path: .phpunit.result.cache
+          key: php${{ matrix.php-version }}-phpunit-result-cache-${{ github.ref }}-${{ github.sha }}
```

Comment:
Does that make sense?

![Screen Shot 2020-08-10 at 11 30 09](https://user-images.githubusercontent.com/605483/89769107-e661bc00-dafc-11ea-8fa7-fe710b7d0e6d.png)


💁‍♂️ For reference, see https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context.